[INFO -- 2017-02-26 03:36:48]
Starting flip_value script
version: 2.0
[INFO -- 2017-02-26 03:36:48]
Initialization strings:
[INFO -- 2017-02-26 03:36:48]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-02-26 03:36:48]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 03:36:48]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 03:36:48]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 03:36:48]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 03:36:48]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-02-26 03:36:48]
initSignal:4
[INFO -- 2017-02-26 03:36:48]
endSignal:8
[INFO -- 2017-02-26 03:36:48]
seqSignal:30
[INFO -- 2017-02-26 03:36:48]
faultMode:0
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 03:37:05]
pythonException: invalid frame
[INFO -- 2017-02-26 03:37:05]
Backtrace BEGIN:
#0  pthread_cond_timedwait@@GLIBC_2.3.2 () at ../sysdeps/unix/sysv/linux/x86_64/pthread_cond_timedwait.S:238
#1  0x00007ffff790c5ea in ?? ()
#2  0x00007ffff7fd2d18 in ?? ()
#3  0x0000000002a10a06 in ?? ()
#4  0x0000000000000017 in ?? ()
#5  0x00007ffff7fd2df0 in ?? ()
#6  0x00007ffff70e17d8 in ?? () from /opt/mpss/3.7/sysroots/k1om-mpss-linux/lib64/libc.so.6
#7  0x00007ffff7fd2d14 in ?? ()
#8  0x00007ffff7fd2de0 in ?? ()
#9  0x00007ffff7ff8630 in ?? ()
#10 0x0000000000000008 in ?? ()
#11 0x0000000000000000 in ?? ()

233	in ../sysdeps/unix/sysv/linux/x86_64/pthread_cond_timedwait.S

Backtrace END
Fliping array at pos: 6
Memory content before bitflip:0x7d0x960x860x3d
Fault Mode: Single bit-flip
base address to flip value: 0x7fffffffc858
address max offset: 4
Memory content after  bitflip:0x7d0xd60x860x3d
frame name: L_lud_omp_123__par_loop2_2_5
symbol name: sum
symbol filename: lud_omp.c
symbol line: 130
value: {0.0661117658, 0.0660457239, 0.065980114, 0.0659135655, 0.0658480376, 0.0657820255, 0.0658387914, 0.0656504557, 0.0655849725, 0.0655194968, 0.0654539615, 0.0653886646, 0.0611798465, 0.0611187965, 0.0610576682, 0.0609962009}
value address: 0x7fffffffc840
Type: The type is an array.
Type sizeof: 64
Field name: None
Field Type: A range type, that is, an integer type with bounds.
Field Type sizeof: 1
Field Type range: (0L, 15L)

[INFO -- 2017-02-26 03:37:05]
Fault Injection Successful after 17.0960450172s
[INFO -- 2017-02-26 03:37:12]
event type: exit
[INFO -- 2017-02-26 03:37:12]
exit code: 1
